src/features/events_list/components/EventsListPanel/EventsListPanel.tsx

```diff
@@ -58,7 +58,7 @@ export function EventsListPanel({
       id: EVENT_LIST_CONTROL_ID,
       name: EVENT_LIST_CONTROL_NAME,
       title: i18n.t('event_list.title'),
-      active: false,
+      active: true,
```

Comment:
also minor fix in order to restore old event_list behaviour, not related to main PR topic, sorry